.goreleaser.yaml

```diff
@@ -19,14 +19,8 @@ archives:
     name_template: >-
       {{ .ProjectName }}_
       {{- title .Os }}_
-      {{- if eq .Arch "amd64" }}x86_64
-      {{- else if eq .Arch "386" }}i386
-      {{- else }}{{ .Arch }}{{ end }}
-      {{- if .Arm }}v{{ .Arm }}{{ end }}
-    # use zip for windows archives
-    format_overrides:
-      - goos: windows
-        format: zip
+      {{ .Arch }}{{ with .Arm }}v{{ . }}{{ end }}
```

Comment:
I think this change is wrong... gorelease v1.23.0 is creating archives with following pattern:
<blabla>_Linux_x86_64.tar.gz